<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PDFParserDB</class>
 <widget class="QMainWindow" name="PDFParserDB">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>275</width>
    <height>376</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>275</width>
    <height>376</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>275</width>
    <height>376</height>
   </size>
  </property>
  <property name="windowTitle">
   <string comment="PDFParserDB">PDFParserDB</string>
  </property>
  <property name="toolTip">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font-family: Times New Roman;
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0 rgba(255, 0, 0, 255), stop:0.427447 rgba(255, 92, 92, 235), stop:1 rgba(255, 179, 179, 255));
</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextBesideIcon</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QListView" name="log">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>251</width>
      <height>161</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QPushButton" name="button_go">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Go!</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>243</width>
      <height>156</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_menu">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_host">
       <item>
        <widget class="QLabel" name="label_host">
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 30); 
border: 1px solid rgba(255,255,255,40);
border-radius: 7px;</string>
         </property>
         <property name="text">
          <string>Host</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_nameDB">
       <item>
        <widget class="QLabel" name="label_nameBD">
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 30); 
border: 1px solid rgba(255,255,255,40);
border-radius: 7px;</string>
         </property>
         <property name="text">
          <string>NameDB</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_2"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_login">
       <item>
        <widget class="QLabel" name="label_login">
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 30); 
border: 1px solid rgba(255,255,255,40);
border-radius: 7px;</string>
         </property>
         <property name="text">
          <string>Login</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_3"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QFrame" name="horizontalFrame_password">
       <property name="styleSheet">
        <string notr="true">QFrame {
    border: 1px solid black;
    border-radius: 0px;
    margin-right: 0;
}</string>
       </property>
       <layout class="QHBoxLayout" name="password">
        <item>
         <widget class="QLabel" name="label_password">
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255, 255, 255, 30); 
border: 1px solid rgba(255,255,255,40);
border-radius: 7px;</string>
          </property>
          <property name="text">
           <string>Password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="enter_password">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit {
    border: none;
    margin: 0;
    font-size: 10pt;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    border: none;
    margin: 0;
    background-color: transparent;
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="resources/res.qrc">
            <normaloff>:/PDFParcerDB/icons/visible.svg</normaloff>
            <normalon>:/PDFParcerDB/icons/invisible.svg</normalon>:/PDFParcerDB/icons/visible.svg</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_way_to_file">
       <item>
        <widget class="QLabel" name="label_way_to_file">
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 30); 
border: 1px solid rgba(255,255,255,40);
border-radius: 7px;</string>
         </property>
         <property name="text">
          <string>Way to file:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="vybor_PDF">
         <item>
          <widget class="QLineEdit" name="way_to_file"/>
         </item>
         <item>
          <widget class="QToolButton" name="browse">
           <property name="styleSheet">
            <string notr="true">background-color: rgba(255, 255, 255, 30); 
border: 1px solid rgba(255,255,255,40);
border-radius: 7px;</string>
           </property>
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionParsing">
   <property name="text">
    <string>Parsing</string>
   </property>
  </action>
  <action name="actionDBSetting">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>DBSetting</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
